src/otx/algo/detection/base_models/detection_transformer.py

```diff
@@ -95,16 +96,22 @@ def export(
         explain_mode: bool = False,
     ) -> dict[str, Any] | tuple[list[Any], list[Any], list[Any]]:
         """Exports the model."""
-        if explain_mode:
-            msg = "Explain mode is not supported for DETR models yet."
-            raise NotImplementedError(msg)
-
-        return self.postprocess(
+        results = self.postprocess(
             self._forward_features(batch_inputs),
             [meta["img_shape"] for meta in batch_img_metas],
             deploy_mode=True,
         )
 
+        if explain_mode:
```

Comment:
Is it agreed with IAI team? On the Geti side model will return dummy values

Reply:
That's fine for now, XAI for the new models is not expected, however, we have to generate stubs to make sure Geti inference won't fail.